ts/masters/modules/map.src.ts

```diff
@@ -21,4 +22,8 @@ import '../../Series/MapPoint/MapPointSeries.js';
 import '../../Series/MapBubble/MapBubbleSeries.js';
 import '../../Series/Heatmap/HeatmapSeries.js';
 import '../../Extensions/GeoJSON.js';
-import '../../Maps/Map.js';
+import MapChart from '../../Core/Chart/MapChart.js';
+
+(Highcharts as any).MapChart = (Highcharts as any).Map = MapChart;
+(Highcharts as any).mapChart = MapChart.mapChart;
```

Comment:
Here the `Highcharts.Map` assignment has to change, I believe. Like `Highcharts.Map` the factory function `Highcharts.mapChart` does not care, if it is called with the `new` keyword or not.